README.md

```diff
@@ -1,7 +1,214 @@
-#TBA...
+# ApiProvider
 
-## Features and bugs
+### Clean and boilerplate free network requests with only a few lines of code.
 
-Please file feature requests and bugs at the [issue tracker][tracker].
+```dart
+  final request = Request(url: Path("https://someapi/user"), httpMethod: HttpMethod.GET);
 
-[tracker]: https://github.com/tikkrapp/api_provider/issues
+  final response = await apiProvider.request<User>(request: request);
+  print("fetched user :${response.body.name}");
+```
+
+### Features:
```

Comment:
looks kinda crowded here :/ 
Any ideas or smiles that we could add to make this more interesting?

Reply:
I don´t see why this looks crowded here. There are only three points.

Reply:
naaah in general the introduction til this paragraph

Reply:
✨ smiley would be good I think. You could use the bigger headlines (those that come with the separator) like we do with the injector. Then I would have the following structure:

1.  Have some eyecatcher like you already have with a simple code example and a fancy quote + the features
2. Installation
3. Usage where you explain the usage in depth

But most importantly: Use the bigger headlines with separators.

Reply:
Eyecatcher ✅ 
Fancy qutote 🚫 (do you have an idea? me not)
Installation ✅ 
In depth usage ✅ (i guess? Do you miss anything?)

Reply:
The fancy quote that you have at the moment is fine.
I think the headlines are too big. Make them one level smaller.

Reply:
DUUUUDE!